src/iop/demosaic.c

```diff
@@ -1105,9 +2301,9 @@ static void lin_interpolate(float *out, const float *const in, const dt_iop_roi_
    I've extended the basic idea to work with non-Bayer filter arrays.
    Gradients are numbered clockwise from NW=0 to W=7.
  */
-static void vng_interpolate(float *out, const float *const in,
-                            const dt_iop_roi_t *const roi_out, const dt_iop_roi_t *const roi_in,
-                            const uint32_t filters, const uint8_t (*const xtrans)[6], const int only_vng_linear)
+static void vng_interpolate(float *out, const float *const in, const dt_iop_roi_t *const roi_out,
+                            const dt_iop_roi_t *const roi_in, const uint32_t filters,
+                            const uint8_t (*const xtrans)[6], const int only_vng_linear)
```

Comment:
All these formatting changes should be remove from this pull request, don't you think? There's a lot of them and it makes the pull look unnecessarily big IMHO.

Reply:
Heh, yeah. Never run clang-format over the whole file...
Use clang-format post-commit hook to format the diff.